src/assets.ts

```diff
@@ -581,37 +606,40 @@ async function addLaunchJsonIfNecessary(generator: AssetGenerator, operations: A
 
         const programLaunchType = generator.computeProgramLaunchType();
         const launchJsonConfigurations: string = generator.createLaunchJsonConfigurations(programLaunchType);
+        const formattingOptions: FormattingOptions = {
+            insertSpaces: true,
+            tabSize: 4,
+            eol: '\n'
```

Comment:
Is it possible to pull these formatting options from VS Code?

Reply:
I was considering that as well, but the spacing was hardcoded before so I did not go further. I will try it.

Reply:
done